ui/app/components/secret-engine/configure-ssh.hbs

```diff
@@ -3,88 +3,84 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-{{#if @configured}}
-  <div class="box is-fullwidth is-sideless is-marginless" data-test-edit-config-section>
-    <div class="field">
-      <label for="publicKey" class="is-label">
-        Public key
-      </label>
+<form {{on "submit" (perform this.save)}} aria-label="save ssh creds" data-test-configure-form>
+  <div class="box is-fullwidth is-shadowless is-marginless">
+    <NamespaceReminder @mode="save" @noun="configuration" />
+    <MessageError @errorMessage={{this.errorMessage}} />
+    {{#unless @model.isNew}}
```

Comment:
I'm seeing this message even creating the config for the first time -- I wonder if maybe we should direct to the details view after create instead, which eliminates the need for this conditional because this screen is always shown when the user is trying to configure

Reply:
Agreed! The current behavior is to navigate to the same screen, but I'm not a big fan of that. Let me do the suggested and see how it turns out.